### PR TITLE
Bump @actions/github from 5.1.1 to 6.0.0

### PR DESCRIPTION
**Pull request by austinvalle**
Bumps [@actions/github](https://github.com/actions/toolkit/tree/HEAD/packages/github) from 5.1.1 to 6.0.0.
<details>
<summary>Changelog</summary>
<p><em>Sourced from <a href="https://github.com/actions/toolkit/blob/main/packages/github/RELEASES.md"><code>@​actions/github</code>'s changelog</a>.</em></p>
<blockquote>
<h3>6.0.0</h3>
<ul>
<li>Support the latest Octokit in <code>@​actions/github</code> <a href="https://redirect.github.com/actions/toolkit/pull/1553">#1553</a>
<ul>
<li>Drop support of NodeJS v14, v16</li>
</ul>
</li>
</ul>
</blockquote>
</details>
<details>
<summary>Commits</summary>
<ul>
<li>See full diff in <a href="https://github.com/actions/toolkit/commits/HEAD/packages/github">compare view</a></li>
</ul>
</details>

### CODE REVIEW

**Review by austinvalle — COMMENTED** [review]
This PR needs to wait until #346 is merged, which will update the latest supported node version to node20

**Review by bendbennett — APPROVED** [review]
LGTM 🚀

**Pull request merged**
